Main/Source/god.cpp

```diff
@@ -570,3 +574,42 @@ void god::ApplyDivineTick()
   if(LastPray > -1 && LastPray < 336000)
     ++LastPray;
 }
+
+std::vector<std::pair<int,festring>> god::vFavID;
+
+bool god::Favour(int iWhat, int iDebit)
+{
+  if((Relation-iDebit) < -1000){
```

Comment:
Gods with relation at 0 or less are normally angry with the player, so they should probably punish the player _and_ ignore any pleas.

Reply:
I will commit soon ,could you check this again? I granted a last favour if relation above 0, but will punish and booming warn